### PR TITLE
fix: Add missing footer to home page

### LINKED ISSUE #175: fix: add footer to home
# Issue

Currently, every page except the home page has a footer. This is unconventional and may also impact SEO.

# Solution

Add the same footer to the home page as the other pages too.

# Additional context

It may be the case that the footer has not been added to the home page for some design-specific reasons. In this case, a solution has to be found, as it is unconventional not to include the footer and impacts SEO.

### PR DESCRIPTION
**Pull request by ashirbadsahu**
### Description
closes #175
This PR addresses the issue of the missing footer on the home page of the website.

#### Changes Include:
- Added footer component to the home page.
- Adjusted CSS to ensure proper positioning of the footer, hero image and wave.
- Made the changed components responsive.

Please review and provide any feedback.

### CODE REVIEW

File: src/layout/Hero/HeroImage.svelte [hunk 1]
```diff
@@ -14,10 +14,10 @@
 	}
 
 	.hero-img {
-		overflow: hidden;
+		overflow:hidden;
```
Comment (KTibow) [inline src/layout/Hero/HeroImage.svelte:17]: Irrelevant change, incorrect formatting

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -4,6 +4,7 @@
 	import SocialHost from '$layout/Hero/SocialHost.svelte';
 	import Wave from '$lib/components/Wave.svelte';
 	import Head from '$lib/components/Head.svelte';
+	import FooterHost from '$layout/Footer/FooterHost.svelte';
```
Comment (KTibow) [inline src/routes/+page.svelte:7]: I would suggest to group imports from layout with other imports from layout

**Review by KTibow — COMMENTED** [review]
Also if this PR closes #175, put "closes #175" in the issue description to link it

File: src/layout/Hero/HeroImage.svelte (hunk 1 above)
Comment (ashirbadsahu) [reply src/layout/Hero/HeroImage.svelte:17]: my bad I did this by mistake and while adding the commit, I didn't see it. I'm doing a new PR.

File: src/routes/+page.svelte (hunk 2 above)
Comment (ashirbadsahu) [reply src/routes/+page.svelte:7]: thanks for your advice. I'm implementing it.

File: src/layout/Hero/HeroImage.svelte (hunk 1 above)
Comment (ashirbadsahu) [reply src/layout/Hero/HeroImage.svelte]: Ohh I did that by mistake, my bad. I have implemented the changes now:)

**Review by Ushie — CHANGES_REQUESTED** [review]
On mobile, the wavey line overlaps with the social icons

image

And overall, it looks totally out of place
image
With the line awkwardly being there
image

I think the intended design should be established first, before attempting to add the footer to the homepage

**Comment by Ushie** [comment]
I'll close this as there's currently no design decided, and this PR would no longer be fit for the design

Nevertheless, thank you much for the contribution!

**Pull request closed without merging**